Sources/Datadog/FeaturesIntegration/WebView/WKUserContentController+Datadog.swift

```diff
@@ -27,6 +27,22 @@ public extension WKUserContentController {
         addDatadogMessageHandler(allowedWebViewHosts: hosts, hostsSanitizer: HostsSanitizer())
     }
 
+    /// Disables Datadog iOS SDK and Datadog Browser SDK integration.
+    ///
+    /// Removes Datadog's ScriptMessageHandler and UserScript from the caller.
+    /// _NOTE:_ This method **must** be called when the webview can be deinitialized.
```

Comment:
This won't render properly, better to use
```suggestion
    /// - Note: This method **must** be called when the webview can be deinitialized.
```